lms/static/scripts/postmessage_json_rpc/server/index.js

```diff
@@ -0,0 +1,9 @@
+/** The entry point for the postmessage_json_rpc_server bundle. */
+"use strict";
```

Comment:
"use strict" is not necessary in an ES6 modules as they are always strict. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Strict_mode_for_modules. If you look at the transpiled code for the module you should find that Babel has added this for you.